app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -1472,6 +1539,11 @@ class PluginApi {
    * ```
    **/
   addPostSmallActionClassesCallback(callback) {
+    deprecated(
+      "`api.addPostSmallActionClassesCallback` has been deprecated. Use the value transformer `post-small-action-class` instead.",
+      POST_STREAM_DEPRECATION_OPTIONS
+    );
```

Comment:
Can we just handle this automatically? As above, anything we can do to reduce the workload for themes/plugins would be great

Reply:
Deprecation removed and I added a test